# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/backdrop-filter-blur-across-tiles.yaml
# Commit: c20cf2fa829a
# Full Hash: c20cf2fa829a19cb6b60909d05abcda40e77dd70
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-04-20 09:42:47
# Regressor Bug: 1764005
# File Overlap Count: 1
# Description:
#   Bug 1764005 - Add backdrop-filter primitive support r=gfx-reviewers,lsalzman,nical
#   
#   Add support for backdrop-filter primitive in WR. Each backdrop
#   filter primitive establishes a render task sub-graph. The primitive
#   instance registers itself as a resolve source to receive the
# ==============================================================================

diff -r d5b01209cf78 -r c20cf2fa829a gfx/wr/wrench/reftests/filters/backdrop-filter-blur-across-tiles.yaml
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/wr/wrench/reftests/filters/backdrop-filter-blur-across-tiles.yaml	Wed Apr 20 01:09:09 2022 +0000
@@ -0,0 +1,19 @@
+# Verify that blurs across tile boundaries do correct readbacks
+---
+root:
+  items:
+    - type: stacking-context
+      backdrop-root: true
+      bounds: 20 20 0 0
+      items:
+      - image: checkerboard(2,32,32,16,22)
+        bounds: 0 0 516 644
+      - type: backdrop-filter
+        bounds: 50 50 416 544
+        filters: ["blur(5,5)"]
+      - type: border
+        bounds: [ 50, 50, 416, 544 ]
+        width: [ 1, 1, 1, 1 ]
+        border-type: normal
+        style: solid
+        color: black